src/boot/main.c

```diff
@@ -328,7 +328,7 @@ void thread3_main(UNUSED void *arg) {
         gBorderHeight = BORDER_HEIGHT_EMULATOR;
         gIsVC = IS_VC();
 #ifdef RCVI_HACK
-        VI.comRegs.vSync = 525*4;   
+        VI.comRegs.vSync = 525*20;   
```

Comment:
Idk much about how this works but is there any reason whatsoever you wouldn't want to use `525` as a constant here?

Reply:
pretty sure it will compile to the exact same output

Reply:
My thinking was different versions/regions but idk I guess this is emu only anyway (although it looks like it impacts VC, is that acceptable?)

Reply:
yes it works on vc too

Reply:
Alright I'll approve then